recipes/seabreeze/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "seabreeze" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6f2a82a47c9fc45129967e714a018889a8363b8f8f83869be3bd82cf9e9417b5
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config  # [not win]
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+    - cython
+    - setuptools_scm
+    - pkgconfig
+    - libusb-compat  # [linux]
+    - pytest-runner
+  run:
+    - python
+    - libusb-compat  # [linux]
+    - numpy
+    - pyusb
+    - future
+    - enum34  # [py<34]
+
+test:
+  imports:
+    - seabreeze
+    - seabreeze.cseabreeze
+    - seabreeze.pyseabreeze
+  requires:
+    - pytest<5
```

Comment:
This should have a single space:  `pytest <5`

Reply:
Thanks. Fixed :)